### PR TITLE
docker,dockerd: Updated to 20.10.3

### PR DESCRIPTION
**Pull request by G-M0N3Y-2503**
Maintainer: @G-M0N3Y-2503
Compile tested: x86_x64, VirtualBox, OpenWrt Master
Run tested: x86_x64, VirtualBox, OpenWrt Master

Description:
* Added script to verify/obtain the correct short SHA for a given version
* Minor style update
* Updated docker CLI and the daemon to 20.10.3

### CODE REVIEW

File: utils/dockerd/git-short-commit.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+	exit 1
+}
+
+GIT_URL="${1}"
+if [ ! -n "${GIT_URL}" ]; then
+	error "Git URL not specified"
+fi
+
+GIT_REF="${2}"
+if [ ! -n "${GIT_REF}" ]; then
```
Comment (neheb) [inline utils/dockerd/git-short-commit.sh:19]: ! -n should be -z . Run through shellcheck.

Comment (G-M0N3Y-2503) [reply utils/dockerd/git-short-commit.sh]: Nice catch.
I guess that specific check isn't in version 0.5.0 of shellcheck.  
Is there a recommended version I should use?

Comment (neheb) [reply utils/dockerd/git-short-commit.sh]: latest one you can I guess.

Comment (neheb) [reply utils/dockerd/git-short-commit.sh]: if you're running debian 10, run

sudo apt -t buster-backports install shellcheck

**Pull request merged**